src/reps/array.js

```diff
@@ -19,184 +19,114 @@ const DOM = React.DOM;
  * Renders an array. The array is enclosed by left and right bracket
  * and the max number of rendered items depends on the current mode.
  */
-let ArrayRep = React.createClass({
-  displayName: "ArrayRep",
```

Comment:
I will investigate that.
Maybe what I did in wrapRender isn't enough, I'll check.
It would be nice to still have these warnings